src/Home.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import '../public/css/Home.css';
+import Service from './components/Service';
+
+class Home extends Component {
+	render() {
+		return (
+			<div>
+				<section className="Home">
+					<nav className="Menu">
+						<a href="#">
+							<img src="images/ironhack-logo.svg" />
+						</a>
+						<a href="#">
+							<img src="images/menu-top.svg" />
+						</a>
+					</nav>
```

Comment:
Lo divertido de React es que podes abstraer lo que te parezca en Componentes, por ejemplo Navbar

```
function Navbar({ images }) {
  return (
     <nav className="Menu">
        { images.map(image => <a href="#">
							<img src={image} />
						</a>) }
    </>
  )
}
```

in Home:

`<Navbar images={[images/ironhack-logo.svg", "images/menu-top.svg"]}`

Reply:
No le veu la utilidad de pasar las imágens com props, si son estáticas...
Si fuera un menú dinámico, por permisos por ejemplo, lo entendría más.

Reply:
Si si, correcto, no es el mejor ejemplo.
Queria ilustrar la posibilidad de hacer componentes, nada mas.
Aun asi, por lo general Navbar suele ser un componente ya que lleva mas que algunas imagenes estaticas